### DISCUSSION TITLE
import data from notion

### DISCUSSION THREAD

**Discussion by Angel0726** [AppFlowy-IO/AppFlowy#466] [General help wanted]
As a  notion user, I want to import my Notion into AppFlowy. So do you have any plan of it?

**Comment by lawweiliang** [comment]
Hi @Angel0726,

People are discussing this issue over github issue section. It is fall under "Future Request".  
Feel free to take a look.  😃 
https://github.com/AppFlowy-IO/AppFlowy/issues/20
